how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -25,8 +25,30 @@ Please refer to ACRN [documentation](https://projectacrn.github.io/latest/index.
 
 This document requires the presence of the ACRN hypervisor and Kata Containers on your system. Install using the instructions available through the following links:
 
-- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
-- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).This guide was tested using KBL-NUC(NUC7i7DNH)
+  > **Note:** Please make sure to have a minimum of 4 logical processors (HT) or cores.
+- ACRN [software](https://projectacrn.github.io/latest/tutorials/kbl-nuc-sdc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- For networking, ACRN supports either MACVTAP or TAP. If MACVTAP is not enabled in the Service OS, please follow the below steps to update the kernel,
```

Comment:
We tend to use colons rather than commas to introduce code blocks (here and below).

Reply:
done.